src/renderer/component/publishForm/view.jsx

```diff
@@ -227,6 +229,7 @@ class PublishForm extends React.PureComponent<Props> {
       price,
       uri,
       channel,
+      isStillEditing
```

Comment:
It looks like the code in this section could be more succinct by just directly copying the desired properties from `this.props` directly to `publishParams` (that don't need to be modified/checked).